community/server/src/docs/ops/server-configuration.asciidoc

```diff
@@ -3,15 +3,14 @@
 
 .Quick info
 ***********
-* The server's primary configuration file is found under _conf/neo4j-server.properties_.
-* Low-level performance tuning parameters and configuration of legacy indexes and the remote shell are found in _conf/neo4j.properties_.
+* Neo4j's configuration is found in _conf/neo4j.properties_.
```

Comment:
It seems like the advice here would not apply to most actual packaged variants of Neo4j, or? Perhaps this should say something like "the location of the Neo4j config file depends on how you installed Neo4j, common locations are blah blah and you can control which location is used via blah blah mechanism"